e2e/README.md

```diff
@@ -0,0 +1,22 @@
+# e2e tests
+
+The `e2e` tests for the action is run by running the action every day on a cron for different use cases. The action that run points to `@main` which helps in catching issues sooner.
+
+If these actions fails to run these actions would create an issue in the repository using https://github.com/naveensrinivasan/Create-GitHub-Issue
```

Comment:
Use an action from the ActionEcosystem for doing this - https://github.com/actions-ecosystem/action-create-issue?

Fyi, there is also a Slack notifier action in the ActionEcosystem can be setup - https://github.com/actions-ecosystem/action-create-issue

Reply:
I tried all of them to create issues and they were having trouble and that is the reason for using this.

Reply:
Interesting. We can use this tool then, LGTM.